### PR TITLE
version 2

### PR DESCRIPTION
**Pull request by ashot1arzumanyan**

### CODE REVIEW

**Review by Rosehair — CHANGES_REQUESTED** [review]
The ./docs folder should stay for the website to be able to deploy from it.
Add instructions in README on how to update production. (yarn build yarn export -o docs).
Note, CNAME and .nojekyll files should not be deleted during yarn export.

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-__pycache__/
-*.pyc
-.DS_Store
+node_modules
+
+/.next/
+/out/
+next-env.d.ts
+tsconfig.tsbuildinfo
+**/*.DS_Store
```
Comment (karenmur1987) [inline .gitignore:12]: add empty line,

File: docs/_next/static/K5VsqeC7NyVSClbmKrPIN/_ssgManifest.js [hunk 2]
```diff
@@ -0,0 +1 @@
+self.__SSG_MANIFEST=new Set,self.__SSG_MANIFEST_CB&&self.__SSG_MANIFEST_CB();
```
Comment (karenmur1987) [inline docs/_next/static/K5VsqeC7NyVSClbmKrPIN/_ssgManifest.js:1]: add empty line,

File: docs/site.webmanifest [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "src": "/android-chrome-512x512.png",
+      "sizes": "512x512",
+      "type": "image/png"
+    }
+  ],
+  "theme_color": "#EEBD20",
+  "background_color": "#F8F8F8",
+  "display": "standalone"
+}
```
Comment (karenmur1987) [inline docs/site.webmanifest:19]: add empty line,

File: pages/team_v2.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import Team from "../src/components/Team/Team";
+
+const TeamPage = () => {
+  return (
+    <Team />
+  );
+};
+
+export default TeamPage;
```
Comment (karenmur1987) [inline pages/team_v2.tsx:9]: add empty line,

File: public/site.webmanifest [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "src": "/android-chrome-512x512.png",
+      "sizes": "512x512",
+      "type": "image/png"
+    }
+  ],
+  "theme_color": "#EEBD20",
+  "background_color": "#F8F8F8",
+  "display": "standalone"
+}
```
Comment (karenmur1987) [inline public/site.webmanifest:19]: add empty line,

**Pull request closed without merging**